File: problem_eight.js
// squre numbr check
var value = 64
function squreNumber(num){
    var sqr = Math.sqrt(num)
    var sq = Math.floor(sqr)
    if(sq*sq == value){
        console.log("yes")
    }
    if(sq*sq != value){
        console.log("no")
    }
}
squreNumber(value)

File: problem_eight.py
# checking squre numbr
import math
Value = int(input("Enter the number:"))
def squrenumber(val):
    sqr =math.sqrt(val)
    sq = math.floor(sqr)
    if sq*sq == val:
        print("yes")
    else:
        print("no")    
squrenumber(Value)        








